mindmeld/core.py

```diff
@@ -124,6 +124,10 @@ def shift(self, offset):
         """
         return Span(self.start + offset, self.end + offset)
 
+    def has_overlap(self, other):
+        """Determines whether two spans overlap."""
+        return self.end >= other.start and other.end >= self.start
```

Comment:
I guess this is on type of overlap, the others being one span is inside the other span length. For this case, does it break the operations of the caller or is this type of overlap fine?

Reply:
This function covers the case where one span is inside the other. Example:

span_one = (1,10), span_two(5,8)
10 >= 5: span_two starts before span_one ends
8 >= 1: span_two starts after span_one starts